sdk/src/wallet/account/operations/transaction/high_level/burning_melting/mod.rs

```diff
@@ -29,38 +25,10 @@ impl Account {
     ) -> crate::wallet::Result<Transaction> {
         let burn: Burn = burn.into();
         let mut options: TransactionOptions = options.into().unwrap_or_default();
-        let mut all_outputs: Vec<Output> = Default::default();
-        let mut all_inputs: Vec<OutputId> = Default::default();
+        options.burn = Some(burn.clone());
```

Comment:
Clone shouldn't be necessary, also can this be just `Some(burn.into())` and `let burn: Burn = burn.into();` removed?